test/hotspot/jtreg/serviceability/sa/ClhsdbLauncher.java

```diff
@@ -144,6 +144,9 @@ private String runCmd(List<String> commands,
         System.out.println("Output: ");
         System.out.println(output);
 
+        // Make sure no -Xcheck:jni failures. WARNING: JNI local refs
+        oa.shouldNotMatch("^WARNING: JNI local refs:.*$");
+        oa.shouldNotMatch("^WARNING in native method:.*$");
```

Comment:
It's not clear to me why this matcher is being done without any tests using `-Xcheck:jni`. Is this intentional?

Reply:
Although no Clhsdb test is explicitly always runs with `-Xcheck:jni`, the option can be added to any test run. For example:

`$ make test  TEST=open/test/hotspot/jtreg/serviceability/sa/ TEST_VM_OPTS="-Xcheck:jni"`

This matching is done to make sure there are no warnings when the test is run in this manner. The matching could be conditional on the test being run with  `-Xcheck:jni`, but I don't really see to point in cluttering up the logic with such a check since the warnings should never appear, whether or not you are using `-Xcheck:jni`.

Reply:
OK, I thought so. Could you expand on the comment saying so, please? Suggestion: `Make sure there are no -Xcheck:jni warnings as -Xcheck:jni might be set via TEST_VM_OPTS on test runs`.

Reply:
Ok, the comment has been updated. Thanks!